hack/jenkins/e2e-runner.sh

```diff
@@ -188,14 +188,18 @@ function get_gci_k8s_version() {
 }
 
 # GCI specific settings.
-# Assumes: JENKINS_GCI_IMAGE_FAMILY
+# Assumes: JENKINS_GCI_IMAGE_FAMILY & JENKINS_GCI_PROJECT
 function setup_gci_vars() {
```

Comment:
These are specific to GCI's continuous integration - if you just want to verifies that k8s HEAD works on GCI, just set `KUBE_NODE_OS_DISTRIBUTION` to `gci` and a GCI image will chose for you. And that behavior is what a user will see if they start a cluster with GCI on nodes.